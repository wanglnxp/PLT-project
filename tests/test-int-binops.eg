int main(){
	int x=5;
	int y=3;
	print(x+y);
	print(x-y);
	print(x*y);
	print(x/y);
	print(x+y);
	print(x==y);
	print(x<=y);
	print(x>=y);
	print(x<y);
	print(x>y);
	print(x!=y);
	return 0;
}

/*comparator*/