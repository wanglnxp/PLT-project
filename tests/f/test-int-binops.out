8
2
15
1
8
false
false
true
false
true
true
